radar_parlamentar/plenaria/tests/tests_ordenacao.py

```diff
@@ -27,27 +27,27 @@ def tearDownClass(cls):
 	def test_ordem_dos_parlamentares(self):
 		proposicao = models.Proposicao.objects.get(id=1)
 		resultado = ordenacao.ordem_dos_parlamentares(proposicao)
-		self.assertEquals(resultado[0][0].descricao, 'Reforma agrária')
-		self.assertEquals(resultado[0][1][0].nome, 'Pierre')
+		self.assertEqual(resultado[0][0].descricao, u'Reforma agrária')
```

Comment:
Sim, porém o resultado vem de dados utilizados pelo módulo importadores, que não terminamos a migração. Dessa forma, o retorno do "resultado[0][0].descricao" ainda é " u'Reforma agrária' " quando o módulo importadores for migrado o teste deve ser atualizado e ter o 'u' retirado. Temos duas alternativas: 1: aceitar o pull request agora e quando importadores for migrado, atualizar o teste. 2: primeiramente, aceitar o pull request de importadores e depois aceitar o da plenária, com o teste já corrigido. Eu particularmente, acho melhor a 2 opção ;)